pkg/where-resolver/reconcile_on_edgeplacement.go

```diff
@@ -87,15 +86,14 @@ func (c *controller) reconcileOnEdgePlacement(ctx context.Context, epKey string)
 	singles := []edgev2alpha1.SinglePlacement{}
 	for _, loc := range locsFilteredByEp {
 		// 2)
-		lws := logicalcluster.From(loc)
 		stsInLws, err := c.synctargetLister.List(labels.Everything())
 		if err != nil {
-			logger.Error(err, "failed to list SyncTargets in Location workspace", "locationWorkspace", lws.String())
+			logger.Error(err, "Failed to list SyncTargets in local cache")
 			return err
 		}
 		stsSelecting, err := filterStsByLoc(stsInLws, loc)
```

Comment:
#1265 should have made this more discriminating, it now needs to reject matches between ISes. @waltforme

Reply:
Just want to confirm I got it, we don't want a location in space A to select a synctarget from space B where A != B. Right?

Reply:
right

Reply:
done